metrics/web/docs/features/deep_links/01_deep_links_analysis.md

```diff
@@ -0,0 +1,106 @@
+# Deep Links Design
+> Feature description / User story.
+
+As a user, I want to pass direct links to project and/or project groups, so that other users can navigate easier.
+
+# Analysis
+> Describe the general analysis approach.
+
+The `Deep Linking` feature improves the navigation experience by allowing users to navigate to a specific resource of an application and to save the application state in a URL.
+
+The analysis stage aims the following goals:
+1. Define the detailed requirements for the deep linking feature;
+2. Investigate existing implementation approaches for the deep linking in the Flutter Web application.
+3. Select the implementation approach for the Metrics Web application that will satisfy the requirements.
```

Comment:
I believe we are not choosing the implementation approach during the analysis. The analysis goal is to investigate any available solutions and understanding if they are suitable for us or not. So, we should overview any existing solutions here (if any) and choose whether we are going to use the existing one, or develop our own approach suitable for our requirements.